pubkey.go

```diff
@@ -45,26 +59,68 @@ func (p *PubKey) loadFile(filename string) {
 		log.Fatalln("Can't read settings file")
 	}
 
-	var u []user
-	if yaml.Unmarshal(d, &u) != nil {
+	s := yamlSetting{}
+	if yaml.Unmarshal(d, &s) != nil {
 		log.Fatalln("Can't unmarshal settings file")
 	}
 
-	p.users = make(map[string]*user, len(u))
-	for i, _ := range u {
-		p.users[u[i].Id] = &u[i]
+	p.users = make(map[string]*user)
+	for i, _ := range s.Users {
+		u := &user{s.Users[i].Id, nil}
+		p.users[u.Id] = u
+	}
+	for i, _ := range s.Teams {
+		t := s.Teams[i]
+		users := p.GetMembers(t.Id)
+		for i := range users {
+			u := &user{users[i], nil}
+			p.users[users[i]] = u
+		}
+	}
+}
+
+func (p *PubKey) GetMembers(team string) []string {
+	users, resp, err := p.client.Organizations.ListMembers(context.Background(), team, nil)
+	if err != nil {
+		log.Println(resp)
+		log.Fatalln("Failed to fetch members: " + team)
```

Comment:
Just in case to make sure, `log.Fatalln` or `log.Fatal**` will cause panic; did you intended?

Reply:
no... `log.Fatal**` triggers `os.Exit(1)`... which anyways stops program.

Reply:
Oh, I didn't know that. I'm going to replace `log.Fatalln` with `log.Println` to continue processes.